demo/top_down_img_demo.py

```diff
@@ -50,6 +50,10 @@ def main():
 
     img_keys = list(coco.imgs.keys())
 
+    # optional
+    return_heatmap = False
+    output_layer_names = ('backbone', )  # default : None
```

Comment:
the returned backbone feature is not used in this code right? 
how about change it to
```python
output_layer_names = None  # e.g. use ('backbone', ) to return backbone feature
```

Reply:
Agreed